gapis/api/vulkan/replay.go

```diff
@@ -901,9 +920,15 @@ func (a API) Replay(
 	}
 
 	if issues != nil {
+		// transforms.Add(newQueryTimestamps(ctx, c))
```

Comment:
Please avoid leaving in commented-out code.

Reply:
fixed